src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -83,12 +82,31 @@ public TileEntity createNewTileEntity(World world, int meta)
         return new TileEntityColonyBuilding();
     }
 
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)